geometric/optimize.py

```diff
@@ -321,7 +321,8 @@ def calcEnergyForce(self):
                     logger.info("Cartesian Hessian is stored in %s/hessian/hessian.txt.\n" % self.dirname)
                     raise HessianExit
                     # sys.exit(0)
-            elif hasattr(self.params, 'hess_data') and self.Iteration == 0:
+            elif hasattr(self.params, 'hess_data'):
```

Comment:
Why was the `self.Iteration == 0` check deleted here?

Reply:
It's because it is already in `elif self.Iteration == 0:` statement. I thought that `self.Iteration == 0` is redundant.

Reply:
Thanks, I think that's right.